src/components/dashboard/Claim.js

```diff
@@ -80,11 +80,22 @@ const Claim = props => {
   // format number of people who did claim today
   /*eslint-disable */
   const formattedNumberOfPeopleClaimedToday = useMemo(
-    () => numeral(numberOfPeopleClaimedToday).format('0a'),
+    () => numeral(numberOfPeopleClaimedToday).format('0a').toUpperCase(),
```

Comment:
to correspond our agreements most correct format could be 0.[0]a
but it doesn't cover all cases. so we have to select format depending of the number we formatting (just in a few particular cases)

pls use this unit test to check your formatting function:

```javascript
const numeral = require('numeral')

describe('Numeral library tests', () => {
  const fixture = {
    0: '0',
    1: '1',
    11: '11',
    99: '99',
    999: '999',
    1000: '1k',
    1100: '1.1k',
    1120: '1.1k',
    1150: '1.2k',
    9900: '9.9k',
    10000: '10k',
    11100: '11.1k',
    11110: '11.1k',
    11150: '11.2k',
    99000: '99k',
    99900: '99.9k',
    100000: '100k',
    110000: '110k',
    115000: '115k',
    115500: '116k',
    999000: '999k',
    999500: '1m',
    1000000: '1m',
    1100000: '1.1m',
    1110000: '1.1m',
    1150000: '1.2m',
    10000000: '10m',
    10100000: '10.1m',
    10110000: '10.1m',
    10150000: '10.2m',
    100000000: '100m',
    110000000: '110m',
    115000000: '115m',
    115500000: '116m',
    999000000: '999m',
  }

  for (let [number, formatted] of Object.entries(fixture)) {
    test(`${number} should be formatted as '${formatted}'`, () => {
      expect(numeral(number).format('0.[0]a')).toBe(formatted)
    })
  }
})
```

also you could add it to the tests

Reply:
```
 Numeral library tests › 115500 should be formatted as '116k'

    expect(received).toBe(expected) // Object.is equality

    Expected: "116k"
    Received: "115.5k"

      42 |   for (let [number, formatted] of Object.entries(fixture)) {
      43 |     test(`${number} should be formatted as '${formatted}'`, () => {
    > 44 |       expect(numeral(number).format('0.[0]a')).toBe(formatted)
         |                                                ^
      45 |     })
      46 |   }
      47 | })

      at Object.test (numeral.test.js:44:48)

  ● Numeral library tests › 999500 should be formatted as '1m'

    expect(received).toBe(expected) // Object.is equality

    Expected: "1m"
    Received: "999.5k"

      42 |   for (let [number, formatted] of Object.entries(fixture)) {
      43 |     test(`${number} should be formatted as '${formatted}'`, () => {
    > 44 |       expect(numeral(number).format('0.[0]a')).toBe(formatted)
         |                                                ^
      45 |     })
      46 |   }
      47 | })

      at Object.test (numeral.test.js:44:48)

  ● Numeral library tests › 115500000 should be formatted as '116m'

    expect(received).toBe(expected) // Object.is equality

    Expected: "116m"
    Received: "115.5m"

      42 |   for (let [number, formatted] of Object.entries(fixture)) {
      43 |     test(`${number} should be formatted as '${formatted}'`, () => {
    > 44 |       expect(numeral(number).format('0.[0]a')).toBe(formatted)
         |                                                ^
      45 |     })
      46 |   }
      47 | })

      at Object.test (numeral.test.js:44:48)
```

Reply:
The following thing fails - according to the mathematical rules, .5 should be rounded to upper
numeral doesn't performs this and adds .5 to the end

Reply:
@sirpy told to use the formatter that was used before - what I did. @serdiukov-o-nordwhale So should we really apply the logic you described.